<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Parameters</class>
 <widget class="QWidget" name="Parameters">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>144</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSpinBox" name="kernelRankSpin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Rank of random feature</string>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_randtype">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Random</string>
   </property>
  </widget>
  <widget class="QLabel" name="svmTypeLabel">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>70</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>C</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_rank">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Rank</string>
   </property>
  </widget>
  <widget class="QComboBox" name="randomTypeCombo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Feature type</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Fourier</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="svmCSpin">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Training Parameter
C-SVM: C cost function, (alpha_i &lt;= C)
Nu-SVM: nu, ratio on the amount of alphas that can be selected
RVM: eps, stopping criterion (the smaller, the more accurate)
Pegasos: lambda, accuracy-generalization tradeoff parameter </string>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.001000000000000</double>
   </property>
   <property name="maximum">
    <double>999.990000000000009</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_width">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Width</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="kernelWidthSpin">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>30</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Width of kernel</string>
   </property>
   <property name="decimals">
    <number>3</number>
   </property>
   <property name="minimum">
    <double>0.001000000000000</double>
   </property>
   <property name="maximum">
    <double>999.990000000000009</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_kerneltype">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Kernel</string>
   </property>
  </widget>
  <widget class="QComboBox" name="kernelTypeCombo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Kernel function</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>RBF</string>
    </property>
   </item>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
